### PR TITLE
Add setting to enable copy buttons

### PR DESCRIPTION
**Pull request by jlai**
@nicolas-girod Per https://github.com/jlai/fitness-dashboard/discussions/59 disable the individual copy values buttons by default, since I think most users would rather copy the entire row.

- Only show copy buttons if enabled in settings
- Reformat using prettier

### CODE REVIEW

**Review by nicolas-girod — APPROVED** [review]

**Comment by nicolas-girod** [comment]
In general, do you prefer merging the PR yourself or that I do it after approving?

**Comment by jlai** [comment]
Normally I'd leave it to the author to merge, although if it's simple enough I'd say it's OK to merge after approval

**Pull request merged**
